install/gcp/installer/forseti_server_installer.py

```diff
@@ -217,8 +235,16 @@ def get_rule_default_values(self):
         Returns:
             dict: A dictionary of default values.
         """
-        organization_id = self.resource_root_id.split('/')[-1]
+        if self.composite_root_resources:
+            # split element 0 into type and id
+            rid = self.composite_root_resources[0].split('/')[-1]
+
+            organization_id = gcloud.lookup_organization(rid)
```

Comment:
the signature of this function is lookup_organization(project_id) so it expects a project_id. What if the first element of the composite root resource is an org or folder?

Could we instead use the value of --project or gcloud config get project instead?

Reply:
I've changed this to be lookup_organization(resource_id, type='projects')

Reply:
This func is used for more lookups than the installation project, and therefore needed to support passing the resource to be inventoried.